packages/react-components/react-dialog/src/stories/Dialog/DialogResponsiveness.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import { Dialog, DialogTrigger, DialogSurface, DialogTitle, DialogBody, DialogActions } from '@fluentui/react-dialog';
+import { Button } from '@fluentui/react-components';
+
+export const ScrollingLongContent = () => {
```

Comment:
```suggestion
export const Responsiveness = () => {
```

nit: to match filename

Reply:
Or rename the file 🐱